.github/workflows/cypress.yml

```
@@ -0,0 +1,130 @@
+        uses: cypress-io/github-action@v6
+        with:
+          working-directory: frontend
+          start: npm run serve
+          wait-on: 'http://localhost:4321'
+          wait-on-timeout: 120
+          config-file: cypress.config.ts
+          install: false
+        env:
+          CYPRESS_BASE_URL: http://localhost:4321
+          CYPRESS_API_URL: http://127.0.0.1:4000/graphql
+          REACT_APP_BACKEND_WEBSOCKET_URL: ws://127.0.0.1:4000/graphql
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)